app/views/MakingALateAppealPage.scala.html

```diff
@@ -25,7 +26,13 @@
 
 @(form: Form[_], pageMode: PageMode)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@layout(pageTitle = messages("makingALateAppeal.headingAndTitle"),
+@layout(pageTitle = if(!request.session.get(SessionKeys.doYouWantToAppealBothPenalties).isDefined) {
+                            messages("makingALateAppeal.headingAndTitle")
+                        } else if (request.session.get(SessionKeys.doYouWantToAppealBothPenalties).contains("yes")) {
+                            messages("makingALateAppeal.headingAndTitle.multi")
+                        } else {
+                        messages("makingALateAppeal.headingAndTitle.first")
+                    },
     isPageFullWidth = false,
```

Comment:
This is a bit repetitive and can be split out into 'reusable blocks' like this: https://github.com/hmrc/penalties-frontend/blob/a968ff76f366c99d08a71c990f60ef3985925a27/app/views/IndexView.scala.html#L40 you can then plug it into wherever it's needed